client/my-sites/post-type-filter/index.jsx

```diff
@@ -30,11 +29,8 @@ import AuthorSegmented from './author-segmented';
 import Button from 'components/button';
 import Gridicon from 'gridicons';
 
-const PostTypeFilter = createReactClass( {
-	displayName: 'PostTypeFilter',
-	mixins: [ UrlSearch ],
-
-	propTypes: {
+export class PostTypeFilter extends Component {
```

Comment:
missing a `static displayName = 'PostTypeFilter'`

Reply:
Fixed up and folded back in via a rebase :)

I always thought we inferred the `displayName` from the declaration. 
Turns out that we don't, so I'm not sure where I got this from.
Anyway, we do get `name` (`this.constructor.name`) from the declaration, which is enough to give us the info we need for debugging etc.:

```
The above error occurred in the <PluginsBrowser> component:
    in PluginsBrowser (created by Localized(PluginsBrowser))
    in Localized(PluginsBrowser) (created by UrlSearch(Localized(PluginsBrowser)))
```

(From `PluginsBrowser` before adding `static displayName ...`)